episodes/03-explore-data.Rmd

```diff
@@ -152,27 +158,45 @@ head(year_country_gdp)
 
 ```
 
-First we define data set, then - with the use of pipe we pass it on to the `select()` function. This way we can chain multiple functions together, which we will be doing now. 
+First we define data set, then - with the use of pipe we pass it on to the `select()` function. 
+This way we can chain multiple functions together, which we will be doing now. 
 
 ## Filter
 
-We already know how to select only the needed columns. But now, we also want to filter the rows of our data set via certain conditions with `filter()` function. Instead of doing it in separate steps, we can do it all together. 
+We already know how to select only the needed columns. 
+But now, we also want to filter the rows of our data set via certain conditions
+with `filter()` function. Instead of doing it in separate steps, we can do it all together. 
 
 In the `gapminder` data set, we want to see the results from outside of Europe for the 21st century. 
-```{r filter-data}
+
+```{r filter-data-and}
 year_country_gdp_euro <- gapminder %>%
-  filter(continent != "Europe" & year >= 2000) %>%
+  filter(continent != "Europe" & year > 2000) %>%
   select(year, country, gdpPercap)
 # '&' operator (AND) - both conditions must be met
 
 head(year_country_gdp_euro)
 ```
 
+Let's now focus only on North American countries  
+```{r filter-data-or}
+year_gdp_namerica <- year_country_gdp_euro %>%
+  filter(country == "Canada" |  country =="Mexico" | country == "United States") 
+
+# '|' operator (OR) - at least one of the conditions must be met
+
+head(year_gdp_namerica)
+```
+
 ::: challenge
 
 ##  Challenge: filtered data frame
 
-Write a single command (which can span multiple lines and includes pipes) that will produce a data frame that has the values for life expectancy, country and year, only for Eurasia. How many rows does your data frame have and why? 
+Write a single command (which can span multiple lines and includes pipes) 
+that will produce a data frame that has the values for **life expectancy**, 
+**country** and **year**, only for Europe **or** Asia. 
```

Comment:
I like "Eurasia" in the previous version because one has to figure out that **or** is needed.